packages/paste-core/components/inline-code/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {InlineCode} from '../src';
+
+describe('InlineCode', () => {
+  it('should render', () => {
+    const {getByText} = render(<InlineCode>sid</InlineCode>);
+    expect(getByText('sid')).toBeDefined();
+  });
```

Comment:
thought: we can remove this test because the component presumably renders if any of the other tests pass